editor/inspector/assets/material.js

```diff
@@ -56,6 +70,25 @@ exports.methods = {
         this.dirtyData.uuid = '';
     },
 
+    async updateCustomInspector() {
+        this.$.customPanel.hidden = false;
+        this.$.section.hidden = true;
+        this.$.materialDump.hidden = true;
+        try {
+            const relatePath = normalize((await this.getCustomInspector()).replace('packages://', ''));
+            const name = relatePath.split(sep)[0];
+            const packagePath = Editor.Package.getPackages({ name, enable: true })[0].path;
+            const path = join(packagePath, relatePath.split(name)[1]);
+            if (this.$.customPanel.getAttribute('src') !== join(path)) {
+                this.$.customPanel.setAttribute('src', path);
+            }
+            this.$.customPanel.update(this.material, this.assetList, this.metaList);
+        } catch (error) {
+            console.error(error);
+            console.error(Editor.I18n.t('ENGINE.assets.material.fail-to-load-custom-inspector', { effect: this.material.effect }));
```

Comment:
error case should inclued packagePath is not exist.

Reply:
The error was expected